Formula/ehco.rb

```diff
@@ -0,0 +1,42 @@
+class Ehco < Formula
+  desc "Network relay tool and a typo :)"
+  homepage "https://github.com/Ehco1996/ehco"
+  url "https://github.com/Ehco1996/ehco/archive/refs/tags/v1.0.6.tar.gz"
+  sha256 "326c468c3790ad01031e52ccb4efcfa5e331d2198fdb13137749c67e8eaacf38"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "cmd/ehco/main.go"
+  end
+
+  test do
+    version_info = shell_output("#{bin}/ehco -v")
+    assert_match(/ehco version */, version_info)
+
+    help_info = shell_output("#{bin}/ehco -h > help.txt")
+    assert_predicate testpath / "help.txt", :exist?
+
+    # run nc server
+    nc_port = free_port
+    nc_pid = fork do
+      exec "nc -l #{nc_port}"
+    end
```

Comment:
```suggestion
    nc_pid = spawn "nc", "-l", nc_port
```

Reply:
i try `nc_pid = spawn "nc", "-l", nc_port` this locally
but i got ` TypeError: no implicit conversion of Integer into String`

Reply:
You can do `nc_port.to_s`.

Reply:
it woks, thanks